documents/emandate.md

```diff
@@ -386,7 +394,7 @@ client.order.create({
 | amount*          | integer | Amount of the order to be paid                                               |
 | currency*        | string  | Currency of the order. Currently only `INR` is supported.                      |
 | receipt         | string  | Your system order reference id.                                              |
-| notes           | object  | A key-value pair       |
+| notes           | object  | A key-value pair                                                             |
 | payment_capture  | boolean  | Indicates whether payment status should be changed to captured automatically or not. Possible values: true - Payments are captured automatically. false - Payments are not captured automatically. |
```

Comment:
payment_capture is mandatory

Reply:
[Added](https://github.com/razorpay/razorpay-python/blob/emandate_doc_correction/documents/emandate.md?plain=1#L398)